test-utils/state-viewer/src/commands.rs

```diff
@@ -52,10 +49,22 @@ pub(crate) fn state(home_dir: &Path, near_config: NearConfig, store: Arc<Store>)
 
 pub(crate) fn dump_state(
     height: Option<BlockHeight>,
+    stream: bool,
+    mut single_file: bool,
+    file: Option<PathBuf>,
     home_dir: &Path,
     near_config: NearConfig,
     store: Arc<Store>,
 ) {
+    assert!(
+        !stream || !single_file,
+        "At most one of options --stream and --single_file can be set"
+    );
+    if !stream && !single_file {
+        println!("Assuming a single file output is requested");
```

Comment:
This logic (with stream and single file coming via two different bools, but then exactly one of them should be true) is a little bit confusing.

Maybe we should get rid of the single_file option and say:

if stream {
    // stuff
} else {
    // put stuff in a single file
}